appveyor.yml

```diff
@@ -153,8 +153,8 @@ build_script:
   - cmake --build ./ -j 4 --config Release
 
 test_script:
-  - "%PYTHON_ROOT%/Scripts/pip.exe install \"gevent!=1.5a1\" \"pytest<5\" psutil \"six>=1.10\" \"numpy>=1.1\""
-  - cmd: if %PYTHON_VER%==27 "%PYTHON_ROOT%/Scripts/pip.exe" install enum34 futures trollius "zipp>=0.5,<2" "pyparsing<3"
+  - "%PYTHON_ROOT%/Scripts/pip.exe install \"gevent!=1.5a1\" pytest psutil \"six>=1.10\" \"numpy>=1.1\" enum34" 
```

Comment:
Thanks for the fixes to the versions.

Note that `enum34` is only for Python 2.  It is a back-port of the enum code from Python 3.4, so needed on versions of Python < 3.4.